teams/release.toml

```diff
@@ -25,6 +25,9 @@ crater = true
 perf = true
 bors.rust.review = true
 
+[github]
+orgs = ["rust-lang"]
```

Comment:
As a relatively simple additional test of this functionality, could we also add `rust-lang-nursery` here and automatically create the release team there as well?

Reply:
I wanted this PR to be a noop, that's why I chose the release team on `rust-lang`. Do you think we should start right away with making changes?

Reply:
I guess it does make sense to leave this change for later. Maybe we can make that the first test though -- since it's relatively low risk and tests team creation.